woodwork/tests/schema/test_schema.py

```diff
@@ -144,6 +144,24 @@ def test_schema_equality(sample_column_names, sample_inferred_logical_types):
                   table_metadata={'created_by': 'user2'}) != schema_with_metadata
 
 
+def test_schema_equality_standard_tags(sample_column_names, sample_inferred_logical_types):
+    schema = Schema(sample_column_names, sample_inferred_logical_types)
+    no_standard_tags_schema = Schema(sample_column_names, sample_inferred_logical_types,
+                                     use_standard_tags=False)
+
+    # Different standard tags values - using logical types with standard tags
+    assert schema.use_standard_tags != no_standard_tags_schema.use_standard_tags
+    assert schema != no_standard_tags_schema
+
+    # Different standard tags values - no logical types that have standard tags
+    nl_ltypes = {col_name: NaturalLanguage for col_name in sample_column_names}
+    schema.set_types(logical_types=nl_ltypes)
+    no_standard_tags_schema.set_types(logical_types=nl_ltypes)
+
+    assert schema.use_standard_tags != no_standard_tags_schema.use_standard_tags
+    assert schema == no_standard_tags_schema
```

Comment:
Still not sure if different values for `use_standard_tags` should make two schemas (and therefore two table accessors) not equal or whether it should just depend on the actual semantic tag values.

I guess one problem with this current behavior is that you could then make the exact same changes to `schema` and `no_standard_tags_schema` (change a logical type to one with standard tags) and end up with schemas that are not equal.

Reply:
Okay, changed so that we also make sure the `use_standard_tags` values are the same